src/labels/CustomLabels.labels

```diff
@@ -1577,6 +1577,14 @@
         <shortDescription>Open Ended Status cannot be converted to enhanced Recurring Donation format</shortDescription>
         <value>Recurring Donation Open Ended Status &quot;{0}&quot; cannot be converted to the Enhanced Recurring Donations format. You must use one of these values: Open, Closed, or None.</value>
     </labels>
+    <labels>
+        <fullName>RD2_DataMigrationInvalidPlannedInstallments</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Planned Installments cannot be migrated to enhanced Recurring Donation format</shortDescription>
+        <value>Number of Planned Installments must be 1 when Open Ended Status is Closed and Divide By/Multiply By is not populated.</value>
```

Comment:
@npsp-reedestockton - Open Ended Status = Closed just means closed. We're looking for fixed-lengths RDs here? In that case we should be looking for Open Ended Status = None.

Reply:
@sam-knox My understanding is that the issue we are trying to fix is a result of the fact that Open Ended Status = Closed RDs may have been either open or fixed.  We can't calculate amount unless Divide By/Multiply By and Planned installments gives us a way to do that.  @RadSFDeveloper has substantive questions about this PR as well, so I think we need to take the discussion into the work item and get straight on the ACs.

Reply:
@sam-knox  I've updated the error label.